src/components/ResourceForm/UserFormPage.tsx

```diff
@@ -75,22 +74,22 @@ const UserFormPage = ({
 
   const [memberName, setMemberName] = useState<string>('');
   const [email, setEmail] = useState<string>('');
-  const [role, setRole] = useState<AccessLevel>(AccessLevel.SUPERVISOR);
+  const [role, setRole] = useState<number>(2);
```

Comment:
Personally, I think using the role ID numbers are really confusing, because I am not exactly sure what '2' represents unless I look at `AccessLevelSelector.tsx`. That being said, was there a reason why we didn't use an enum to represent the different values in a more readable fashion?